src/main/webapp/resources/js/pages/projects/samples-metadata-import2/components/SampleMetadataImportFileUploader.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from "react";
+import {
+  Alert,
+  Divider,
+  message,
+  Space,
+  Typography,
+} from "antd";
+import { DragUpload } from "../../../../components/files/DragUpload";
+import { setBaseUrl } from "../../../../utilities/url-utilities";
+import { SampleMetadataImportSteps } from "./SampleMetadataImportSteps";
+
+export function SampleMetadataImportFileUploader() {
+  const { Text, Title, Paragraph } = Typography
+  const options = {
+    multiple: false,
+    showUploadList: true,
+    accept: [".xls", ".xlsx", ".csv"],
+    action: setBaseUrl(`/projects/${window.project.id}/sample-metadata/upload/file`),
```

Comment:
We will move this over to redux-toolkit.  We have a better way of handling the project id.

Reply:
Ok. Should this be changed now or in a future PR?

Reply:
I don't see a reason why not to know.  Take a look at the settings/index.js and settings/store.js.  You will just need to add the projectApi for now.